notebooks/data-sources/TestGrid/metrics/blocked_timed_out.ipynb

```diff
@@ -2,67 +2,126 @@
  "cells": [
```

Comment:
<p>You have already limited the dataset to 100000, so I don't think these <code>*.head(10000)</code>  's are required here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Fixed :+1: